src/WebJobs.Script.WebHost/Home.html

```diff
@@ -284,7 +284,7 @@
 
         <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAKgAAAAXCAYAAACf1wQUAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAABJNJREFUeNrsWo1xnTAMzut1AEagGzCCuwHdgA3CBmED0glIJ3gj8DZwN3A2IBtQ0zM9RSfJEu/nmjx053st2MaWPsnS5zw87LLLfyyHXQXXl3meq/hTx/Z2OByed41sU6IDrTSOLdexxLsxtim2+k71OsxAruUASf/VZ1YklGGrEQjQrzLeITi7tPfFQXurXpXfKICOp3sB6KyNoil6shHiniNo2vciDfO+ie145jdaZLfmXgDaGaPE1Y6wT6BTJ70/8xsBROn5XMB/BGUG7XGRjpdpB6is063vlbnnKq319PsI8pV49jO2PrYFfE2sOl+E8cuxXeSKr790weFwyvQp01y/lxb7v3FzcHMuDhN/qtSW8a94LsrJUv/VqOIYYq2vse8rtWfu/5yOcnoi5DH9Lut9jvM8pXUtx3wnfUekduIaDHZ714ew1apfyl6r3su0h5MmgpYpb1zEK4+XQchB2Six5KVgjplL9sHzCvZHkbzH0RxIz3y/Z/rPTK7HzT8mQ3DpklrOyXGBHYIyjWPXoFkPUxjDORqoMxT5R8rurBMRE+fypxqB2gRQBI4pKXd55rlNA3D6FcQJnB708WvFjMB/FOgfn3Lpf2OEvgGvFTwvbgVQZKOCOPLdRoD2lwAoKp5HgK0KgDYkvXcIsFX2YzA6MosbIX1kASgC9wCjz/peUKwDRwcGepPhIRuCeWiloxCvNQOU4VY56OJwjOMFC1WYHDzgE/MCAB2gbde8GDj1KBTcRw1A2aSb8lQjQL2FG+WYBcQBdsL4dykL4mdz+WF2rYjJKK4NUORgjcC9FgagT9DOFwBooAKPVMhBe2oACiv0jolKQXGU43mhcmsjQJ1lw1SUI4DtubGoX6MEjLsBQFuOaZHAm+FQa+vaMgBthdPMo5tL2LwKoNgbc0YzANRtyLc4gHYaSoyKmESBdKSqby11Q0XyKwI0gNyOMnJQRH0nFZEXAKgTTjJ7Ls58rCTyt46qFG8E0IIB6GgFKIisgajIK+tabwVQlF5ppMzw137r2nJV/CUA+iXDhy383sqDPiHu7dcFiOzC0l/gM51hmlcw30ts3+I/f8R2AnMdCWeorPNfSR6N/alj/pj40kWf33MOYXmulNMhI2qPQVG05xJwQwStrPfGwtxOpCYUOZtQkdconWmVUa2yRlCLo4LI1yuLnyAUdM6aVnF5vTKCijzt5ms5IjQPBhBRm4B8ZnHm2ibpmEc0yqD91gpIsPfArVUAgxagTmmjRuOQHJeNispO6QhDht2wALTJBScWD5mJnSKv2cqDekRvVBhsmbW1qNDBc3mGRvFE5dpiAKC9e5TDlohnrTcCtNUcm5wjCI45AU6yRAQ5V0U7gj9u0Z6PXEGTsVWBcv4WvWvYvSmUGaS/ltlwkzRoqzjF2oZMzj1h42f6D4Lnc9Js0OmovUnCqZYy4uLbMstVZylc7c5bjnjiJkl/m6aYuNnyp2OKu3isuEDwrpqKsiYMvl6hlozxsKK8cPRUOHJw9JRBpwVac1BylqUSoO6c61Zmz166nFHaqrTqcpdddtllF6v8EWAA2wxYm4SN9zUAAAAASUVORK5CYII=" alt="Microsoft Azure" class="azureLogo" />
 
-        <div class="bodyHeadline">Your Function App is up and running</div>
+        <div class="bodyHeadline">Your Function App 2.0 preview is up and running</div>
```

Comment:
Why do we need to do this? I think we should revert this.

Reply:
This just helps us easily identify the version when hitting the home page. We can remove it, but this is helpful. We can also make this a bit smarter so we don't have to remember to remove this (or simply have an issue tracking). We had also discussed returning the version information.

Reply:
Yeah, I think it would be better to return version info or something, so we don't have to track removing this. However, I still don't understand how this is useful. Customers themselves know what version of the product they're using, and we have our own internal means of discovering this, so why does it need to be on the landing page?